src/vs/workbench/contrib/searchEditor/browser/searchEditor.ts

```diff
@@ -482,6 +482,14 @@ export class SearchEditor extends AbstractTextCodeEditor<SearchEditorViewState>
 	}
 
 	async triggerSearch(_options?: { resetCursor?: boolean; delay?: number; focusResults?: boolean }) {
+		const focusResults = this.searchConfig.searchEditor.focusResults;
+
+		if (_options === undefined) {
+			_options = { focusResults: focusResults };
+		} else if (_options.focusResults === undefined) {
+			_options.focusResults = focusResults;
+		}
```

Comment:
This logic is saying "if options don't define `focusResult` field, then use the setting", right? Maybe a comment would make this clearer.